app/models/concerns/operations/stock_operations.rb

```diff
@@ -68,5 +70,98 @@ def register_loss(package, quantity:, from_location:)
         location: from_location
       )
     end
+
+    def pack_or_unpack(params, user_id)
```

Comment:
Recommendation: pass in the params as ruby options arguments, e.g

```
def pack_or_unpack(box_id:, package_id:, :quantity, user_id:)
    ...
end
```

Otherwise we have no idea what we're supposed to pass to this method, or we need to dive deep in the code to figure it out

Reply:
Makes sense. Will do that. 👍

Reply:
Thanks : )